app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <div id="expenses" class="expense-container">
+    <%= render @expenses %>
+  </div>
+</div>
+
+<div class="bg-secondary d-flex fixed-bottom py-3 px-2">
+
+  <div class="container d-flex justify-content-center transaction-container">
+    <%= link_to new_group_expense_path, class: "new-transaction" do %>
+    <p>New transaction</p>
+  </div>
+
```

Comment:
- [ ] I think you didn't update the background color for your transaction nutton as requireed be the previous reviewer  Kindly maintain a consistent design that follows these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for Colors and Typography.

## Your transaction page not category page button here your new transaction button  is different than the given design 
 ### The `new Transaction` button should be the same as your `Add new category` button.
> 
> Your `New Transaction` button:
> 
![image](https://user-images.githubusercontent.com/58553711/221363638-1f37ecb4-d2c9-4438-8fc7-925967496d4f.png)


> 
- The original design
![image](https://user-images.githubusercontent.com/58553711/221363677-86efc135-69a0-4e2f-8c38-f21f61a25631.png)

Reply:
Thank you for the reminder on the color. 😄

Reply:
Done ...